README.md

```diff
@@ -133,15 +133,13 @@ This allows using Jaeger UI to find the trace by this tag.
 
 This library internally uses Zipkin Thrift data model and conventions, 
```

Comment:
this is now out of date since I merged #111. Do you mind rephrasing that only in-band interop with Zipkin is possible with your change?